BootstrapRestauraunt/menu2.html

```diff
@@ -0,0 +1,339 @@
+<!doctype html>
```

Comment:
This page looks very nice. Nevertheless, there are quite a few html errors when running through a validator - 54 errors, to be exact.